### PR TITLE
fix issue with merging parse_options in source_as_blob

### PR DESCRIPTION
**Pull request by catstavi**

### CODE REVIEW

File: socrata/revisions.py [hunk 1]
```diff
@@ -243,7 +243,8 @@ def source_as_blob(self, filename, parse_options = {}):
         """
         Create a source from a file that should remain unparsed
         """
-        return self.create_upload(filename, parse_options.update({'parse_source': False}))
```
Comment (branweb1) [inline socrata/revisions.py]: ah so this was potentially overriding passed in options. ok

Comment (catstavi) [reply socrata/revisions.py]: no :( the method doesn't return anything, just mutates, so I was passing None here.

Comment (branweb1) [reply socrata/revisions.py]: ah man tough to bounce from elixir to imperative-style lang

Comment (catstavi) [reply socrata/revisions.py]: why doesn't it work like that other method I made up....

**Pull request merged**